assets/sass/5_layouts/_layout-f.scss

```diff
@@ -1,12 +1,89 @@
 &.layout-f {
     height: 100%;
 
+    .login-main {
```

Comment:
Could these style be defined in component specific file rather than the top level layout-f? Are they really tied to layout-f?

Reply:
They are all only tied to layout f because we don't have a login/password recovery like this anywhere else. I'll look into it though.